### PR TITLE
Blog post for TypeChat 0.1.0.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
One question: should it be "TypeChat *for TypeScript* 0.1.0"

File: site/src/blog/announcing-typechat-0-1-0.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        .describe("The sentiment of the text")
+});
+
+// Maps the property "SentimentResponse" to the above Zod validator.
+export const SentimentSchema = {
+    SentimentResponse
+};
+```
+
+Note that while TypeScript schema files can use raw JavaScript/TypeScript `// comment` syntax, TypeChat generates schemas from Zod based on [whatever we pass in to `.describe()` calls](https://zod.dev/?id=describe).
```
Comment (ahejlsberg) [inline site/src/blog/announcing-typechat-0-1-0.md:137]: ... TypeChat generates comments ~schemas~ from Zod ...

**Review by ahejlsberg — APPROVED** [review]
Approved with minor fix.

File: site/src/blog/announcing-typechat-0-1-0.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
++ const validator = createTypeScriptJsonValidator<SentimentResponse>(schema, "SentimentResponse");
++ const translator = createJsonTranslator(model, validator);
+
+...
+```
+
+## Zod Validators
+
+The second change builds on pluggable validators: TypeChat makes it possible to create validators from Zod schemas.
+[If you're not familiar with Zod](https://zod.dev/), it's a fairly popular library in the TypeScript/JavaScript ecosystem for validating data.
```
Comment (pierceboggan) [inline site/src/blog/announcing-typechat-0-1-0.md:116]: no need to differentiate between "fairly popular" and "popular". I'd just remove 'fairly'

File: site/src/blog/announcing-typechat-0-1-0.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+One strength of this library is that as Zod type validator objects are constructed, static types can be derived from them.
```
Comment (pierceboggan) [inline site/src/blog/announcing-typechat-0-1-0.md:117]: Isn't the implication of zod that you also can now have dynamic schemas you generate at runtime? I think you're getting at that here but it's worth connecting that dot explicitly for readers

**Review by pierceboggan — CHANGES_REQUESTED** [review]
LGTM, requested a few small comments!

**Pull request merged**